docs/semgrep-cloud-platform/teams.md

```diff
@@ -0,0 +1,181 @@
+# Manage access to projects
+
+Use the **Teams** feature to manage access to resources, such as findings in your projects.
+
+![The Teams tab within the Settings page](/img/access-teams.png)
+**Figure**. The **<i class="fa-solid fa-gear"></i> Settings > Access > Teams** tab displays both top-level teams and subteams.
+
+The **Teams** feature enables admins to grant or limit access to specific projects in Semgrep Cloud Platform (SCP). **Projects** are repositories or codebases you have added to SCP for scanning. You can quickly assign projects to large groups of members by assigning teams and subteams to members of your organization.
+
+This feature helps security engineers and developers in large organizations focus on the projects that are relevant to their specific department or team.
+
+When you limit a member's access to a subset of your projects, their Dashboard and Findings pages all reflect that change. For example, their total finding count is based on the total number of findings of the projects they can access.
+
+This document walks you through the following:
+
+- How to approach team management and project access in Semgrep
+- How to create, view, update, and delete teams and subteams
+- How to assign or unassign projects to teams
+
+## Roles and visibility
+
+A user is any person who has been added to your organization in Semgrep. Users can be administrators (admins), managers, or members.
+
+<dl>
+<dt>Admin</dt>
+<ul><li>A user who has access to all features, resources, and projects of their Semgrep deployment. Admins can also change the role of members and managers.</li>
+<li>By definition, an admin does not need to be assigned to a team to have access to the resources of a project. As such, an admin can't be removed from a team.</li>
+<li>An org admin can change the role of any other admin or user, including a fellow admin.</li></ul>
+<dt>Manager</dt>
+<ul>
+<li>A user who can grant access to projects by creating subteams and assigning members to these subteams.</li>
+<li>A manager role is restricted to the teams where they have been assigned as a manager. Users can be managers of some projects, but members for others. For more information, see the <a href="#the-manager-role">manager role</a>.</li>
+</ul>
+<dt>Member</dt>
+<ul>
+<li>A user who has access to some features, resources, and projects of their Semgrep deployment.</li>
+<li>By default, a member starts out with no access to any project.</li>
+<li>To grant members access to a project and its findings, you must add a member to a team, and that team must be assigned projects.</li>
+<li>Members can scan their local or personal repositories through a personal account.</li>
+</ul>
+</dl>
+
+![A member's view of the Projects page.](/img/access-member-view.png)
+**Figure**. A member's view of the Projects page. It displays projects that are assigned to the team they are a member of, but they cannot edit a project nor can they scan new projects in their organizational account.
+
```

Comment:
This is a completely optional suggestion, but maybe we could add headers to the tables? It would split them up a bit and allow people to link directly to the tables.